CHANGELOG.md

```diff
@@ -1,17 +1,20 @@
-## <small>5.0.4 (2021-02-21)</small>
```

Comment:
Can you remove these changelog changes from the alpha releases?

Reply:
done